docker-compose.e2e-basic.yml

```diff
@@ -82,7 +82,7 @@ services:
       - 9999:1080
 
   transactor:
-    image: mysteriumnetwork/transactor:0.7.0
+    image: mysteriumnetwork/transactor:0.7.2
```

Comment:
Just curious, any reason you picked 0.7.2 instead of .3 or .4?

Reply:
Because... well.. I thought .2 was the latest 🤦